core/src/main/java/org/apache/carbondata/core/datastore/AbstractBlockIndexStoreCache.java

```diff
@@ -92,13 +94,14 @@ protected void checkAndLoadTableBlocks(AbstractIndex tableBlock,
     TableBlockInfo blockInfo = tableBlockUniqueIdentifier.getTableBlockInfo();
     long requiredMetaSize = CarbonUtil.calculateMetaSize(blockInfo);
```

Comment:
any particular reason? we are using the value to determine whether we need to load the block into memory.